qa-pipelines/tasks/lib/azure-aks.sh

```diff
@@ -73,9 +92,14 @@ azure_check_lbs_ready_in_namespace() {
 
 azure_wait_for_lbs_in_namespace() {
     local namespace=$1
+    local lb_name=${2:-""}
     local count=0
-    while ! azure_check_lbs_ready_in_namespace $namespace; do
-        sleep 30
+    while ! azure_check_lbs_ready_in_namespace $namespace $lb_name; do
+        if [[ -z "$lb_name" ]]; then
+            sleep 30
+        else
+            sleep 180
```

Comment:
Why is the timeout different here? Needs a comment at least. (Basing it on "if the lb has a name" is weird too… why isn't it just a separate parameter with a default value?)